stan/math/opencl/kernels/divide_columns.hpp

```diff
@@ -0,0 +1,66 @@
+#ifndef STAN_MATH_OPENCL_KERNELS_DIVIDE_COLUMNS_HPP
+#define STAN_MATH_OPENCL_KERNELS_DIVIDE_COLUMNS_HPP
+#ifdef STAN_OPENCL
+
+#include <stan/math/opencl/kernel_cl.hpp>
+#include <stan/math/opencl/buffer_types.hpp>
+
+namespace stan {
+namespace math {
+namespace opencl_kernels {
+// \cond
+static const char *divide_columns_kernel_code = STRINGIFY(
+    // \endcond
```

Comment:
Why do we want to include the `divide_columns_vec` into the Doxygen doc? (that's a legit question... there might be good merit for having it or leaving it out.)

Reply:
Do you mean for this specific kernel `divide_columns_vec` why do we want it in the doxygen doc? Or is this more about documentation for the kernels in general.

For (1) and (2) I think my answer is just that I'd rather have docs for all the kernels.